src/linker/ref/Linker/OverrideInformation.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Mono.Cecil;
+
+namespace Mono.Linker
+{
+	public class OverrideInformation
+	{
+		internal OverrideInformation ()
```

Comment:
Why do we need to declare an internal .ctor in a ref assembly?

Reply:
There were errors about missing ctor in apidiff tool

Reply:
(Specifically if you don't declare a ctor, the ref assembly will have the implicit public ctor, which doesn't exist in the implementation.)

Reply:
Good to know - thanks!